plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/gitlabMergeRequest.ts

```diff
@@ -42,6 +42,7 @@ export const createPublishGitlabMergeRequestAction = (options: {
     branchName: string;
     targetPath: string;
     token?: string;
+    removeSourceBranch: boolean;
```

Comment:
Think that this should be optional `removeSourceBranch?` and then default it to the default option if needed.

Reply:
Yep just a unit test would be nice to test to check that the prop is getting passed to the api.